plugins/module_utils/mso.py

```diff
@@ -178,6 +178,10 @@ def __init__(self, module):
         # normal output
         self.existing = dict()
 
+        # mso_rest output
+        self.jsondata = None
+        self.error = dict(code=None, text=None)
```

Comment:
text -> message as we only use it with message. We might also add info as that is the 3rd element added there.

Reply:
done